ext/opentelemetry-ext-jinja2/tests/test_jinja2.py

```diff
@@ -0,0 +1,114 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import jinja2
+
+from opentelemetry import trace as trace_api
+from opentelemetry.test.test_base import TestBase
+
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+TMPL_DIR = os.path.join(TEST_DIR, "templates")
+
+
+class TestJinja2Instrumentor(TestBase):
+    def setUp(self):
+        super().setUp()
+        # prevent cache effects when using Template('code...')
+        # pylint: disable=protected-access
+        jinja2.environment._spontaneous_environments.clear()
+
+    def test_render_inline_template(self):
+        template = jinja2.environment.Template("Hello {{name}}!")
+        self.assertEqual(template.render(name="Jinja"), "Hello Jinja!")
+
+        spans = self.memory_exporter.get_finished_spans()
+        self.assertEqual(len(spans), 2)
+
```

Comment:
Is there any parent-child relationship between span[0] and span[1] to be tested?